dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/resources/l10n/com.ibm.websphere.appserver.jwt-1.0.properties

```diff
@@ -14,4 +14,4 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature allows runtime to create JWT tokens.
+description=This feature allows the runtime to create JWTs.
```

Comment:
acronym isn't defined yet on the page so should be

This feature allows the runtime to create JSON web tokens (JWT)

Reply:
Updated